subsys/shell/Kconfig

```diff
@@ -47,7 +47,8 @@ config SHELL_CMD_BUFF_SIZE
 	int "Shell command buffer size"
 	default 256
 	help
-	  Maximum command size.
+	  Maximum command size in bytes. One byte is reserved for end of string
```

Comment:
"reserved for the string terminator character"

Reply:
done